src/camera.py

```diff
@@ -0,0 +1,76 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class.
+
+Uses SunFounder Servo class to control camera servos allowing for pan and tilt.
+Uses SunFounder filedb to configure offset for pan and tilt functions.
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+
+    def __init__(self, bus_number=1):
+        self._DB = filedb.fileDB(getcwd() + "/../SunFounder_PiCar-V/remote_control/remote_control/driver/config")
+        self._pan_offset = int(self._DB.get('pan_offset', default_value=0))
+        self._tilt_offset = int(self._DB.get('tilt_offset', default_value=0))
+        self._pan_servo = Servo.Servo(self.PAN_CHANNEL, bus_number=bus_number, offset=_self.pan_offset)
+        self._tilt_servo = Servo.Servo(self.TILT_CHANNEL, bus_number=bus_number, offset=_self.tilt_offset)
+
+
+    '''
+    Controls the horizontal movement of the camera.Usable inputs are [0,180]. 0 refers to the 
+    vehicles righthand side while 180 refers to the vehicles lefthand side. 
+    '''
+    def pan(self, degree):
+        self._pan_servo.write(self.validate_degree(degree,180))
+
+
+    '''
+    Writes pan offset to a config file within current directory, lower values direct the 
+    camera more toward the vehicles righthand side while greater values direct the camera more t
+    oward the vehicles lefthand side. This file can be changed incrementally with this function
+    or manually in the file system.
+    '''
+    def calibrate_pan(self, degree):
+        new_pan_cali = self._pan_offset + degree
+        self._DB.set('pan_offset', new_pan_cali)
+        self._pan_offset = new_pan_cali
+
+
+    '''
+    Controls the til movement of the camera. Usable inputs are [0,150]. 0 refers to 
+    the vehicles front while 150 refers to the camerafacing upwards.
+    '''
+    def tilt(self, degree):
+        self._tilt_servo.write(self.validate_degree(degree,150))
+
+
+    '''
+    Writes tilt offset to a config file within current directory, lower offset directs the camera
+    more towards the ground while higher offset directs the camera more vertical. This file can
+    be changed incrementally with this function or manually in the files system.
+    '''
+    def calibrate_tilt(self, degree):
+        new_tilt_cali = self._tilt_offset + degree
+        self._DB.set('tilt_offset', new_tilt_cali)
+        self._tilt_offset = new_tilt_cali
+
+
+    '''
```

Comment:
Doesn't return the tilt and pan **_ouptut_**. It validates a value is within [0, max]. Say that. Yes, it's used to validate the tilt and pan degrees are in the right ranges, but that's it's more generic than that.

Also, get rid of the inputs and outputs for now. Describe them in the comments. We'll have a better format for this later.